src/gameState.ts

```diff
@@ -0,0 +1,82 @@
+import { Board } from "./board/board";
+import { loadGameConfigFromFile } from "./gameLoader";
+import { Coordinates, checkIfMoveIsAllowed } from "./move/move";
+import { Player } from "./player";
+import { Tile } from "./tile/tile";
+
+const initialGameState: GameState = {
+  currentPlayer: "black",
+  turnNumber: 1,
+};
+
+export interface GameState {
+  board?: Board;
+  turnNumber: number;
+  currentPlayer: Player;
+  winner?: Player;
+  selectedMoveCordinates?: Coordinates;
+}
+export type GameActionName =
+  | "SELECT_TILE"
+  | "FORFEIT"
+  | "WAITING"
+  | "END_TURN"
+  | "LOAD_BOARD";
```

Comment:
All these do not look like actions to me. E.g. `SELECT_TILE` could be an action but `WAITING` seems like a state (or status) to me.

Also, you are anticipating on states that are out of scope of this PR (like `FORFEIT`). No need to implement them now. Remember that the client can change goals and priorities at any time, so just focus on the current feature you are implementing.

Reply:
No longer exists since merge with https://github.com/marmelab/kamon/pull/8